### DISCUSSION TITLE
WSL closes immediately

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/WSL#8916] [General]
### Version

Microsoft Windows [Version 10.0.19044.2006]

### WSL Version

- [X] WSL 2
- [ ] WSL 1

### Kernel Version

5.10.102.1

### Distro Version

Ubuntu

### Repro Steps

Open Ubuntu

### Expected Behavior

Ubuntu should stay open

### Actual Behavior

Ubuntu closes immediately with error 
`-bash: loadlocale.c:129: _nl_intern_locale_data: Assertion `cnt < (sizeof (_nl_value_type_LC_TIME) / sizeof (_nl_value_type_LC_TIME[0]))' failed.`

**Comment by benhillis** [comment]
Looks like something might be wrong with your .bashrc. I'd suggest running `wsl.exe --exec /bin/sh` to see if you can fx it.

**Reply by ghost** [reply]
It looks like my libc/glibc is broken. I remember I did manual upgrade to newer version and now I am getting this error:
/lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.30' not found (required by /lib/x86_64-linux-gnu/libselinux.so.1)
